api/docs/test_suite.dox

```diff
@@ -315,6 +315,15 @@ The comments at the top of runsuite_ssh.cmake describe additional options.
 
 Unfortunately our test suite is not as clean as it could be.  Some tests can be flaky and while they pass on the machines of the existing developers and on our automated test machines, they may fail occasionally on a new machine.  Please search the issue tracker before filing a new issue to see if a test failure has already been seen once before.  We welcome contributions to fix flaky tests.
 
+Flaky tests are marked in one of two ways:
+
+ - Append "_FLAKY" to the test's name
+    - See suite/tests/CMakeLists.txt for examples
```

Comment:
Let's make these links to the online file viewer, here and below.

Reply:
Done